qa/tasks/cephfs/filesystem.py

```diff
@@ -256,8 +256,11 @@ def _fail_restart(id_):
 
         self._one_or_all(mds_id, _fail_restart)
 
-    def newfs(self, name):
-        return Filesystem(self._ctx, create=name)
+    def newfs(self, _name, create=True):
+        return Filesystem(self._ctx, name=_name, create)
+
+    def newfs_raw(self, name):
+        return Filesystem(self._ctx, create=None, name=name)
```

Comment:
I'd still like to see `newfs_raw` removed. Just pass kwargs through newfs to the Filesystem constructor.